tests/client.test.ts

```diff
@@ -475,6 +477,139 @@ describe("OpenFGA Client", () => {
+              },
+            }),
+            expect.any(Object)
+          );
+
+          mockWrite.mockRestore();
+        });
+
+        it("should pass onMissingDeletes option to API", async () => {
+          const tuple = {
+            user: "user:81684243-9356-4421-8fbf-a4f8d36aa31b",
+            relation: "admin",
+            object: "workspace:1",
+          };
+
+          const mockWrite = jest.spyOn(fgaClient.api, "write").mockResolvedValue({} as any);
+
+          await fgaClient.write({
+            deletes: [tuple],
+          }, {
+            conflict: {
+              onMissingDeletes: OnMissingDeletes.Ignore,
+            }
+          });
+
+          expect(mockWrite).toHaveBeenCalledWith(
+            baseConfig.storeId,
+            expect.objectContaining({
+              deletes: {
+                tuple_keys: [tuple],
+                on_missing: "ignore",
+              },
+            }),
+            expect.any(Object)
+          );
+
+          mockWrite.mockRestore();
+        });
+
+        it("should pass both conflict options to API", async () => {
+          const writeTuple = {
+            user: "user:81684243-9356-4421-8fbf-a4f8d36aa31b",
+            relation: "admin",
+            object: "workspace:1",
+          };
+          const deleteTuple = {
+            user: "user:81684243-9356-4421-8fbf-a4f8d36aa31b",
+            relation: "admin",
+            object: "workspace:2",
+          };
+
+          const mockWrite = jest.spyOn(fgaClient.api, "write").mockResolvedValue({} as any);
+
+          await fgaClient.write({
+            writes: [writeTuple],
+            deletes: [deleteTuple],
+          }, {
+            conflict: {
+              onDuplicateWrites: OnDuplicateWrites.Ignore,
+              onMissingDeletes: OnMissingDeletes.Error,
+            }
+          });
+
+          expect(mockWrite).toHaveBeenCalledWith(
+            baseConfig.storeId,
+            expect.objectContaining({
+              writes: {
+                tuple_keys: [writeTuple],
+                on_duplicate: "ignore",
+              },
+              deletes: {
+                tuple_keys: [deleteTuple],
+                on_missing: "error",
+              },
+            }),
+            expect.any(Object)
+          );
+
+          mockWrite.mockRestore();
+        });
+
+        it("should not include conflict options when not specified", async () => {
+          const tuple = {
+            user: "user:anne",
+            relation: "writer",
+            object: "document:2021-budget",
+          };
+
+          const mockWrite = jest.spyOn(fgaClient.api, "write").mockResolvedValue({} as any);
+
+          await fgaClient.write({
+            writes: [tuple],
+          });
+
+          expect(mockWrite).toHaveBeenCalledWith(
+            baseConfig.storeId,
+            expect.objectContaining({
+              writes: {
+                tuple_keys: [tuple],
+                on_duplicate: undefined,
```

Comment:
This might just be how the JS generator template works but in other languages the default value for each property is being sent (`"error"`), I'm not sure if we should make them aligned (which would require template updates) or just accept the language differences.

Reply:
Good point! Just updated it to send `"error"` as the default for conflict options when they're not specified